uk.co.compucorp.civicrm.tasksassignments/views/contact/documents.html

```diff
@@ -17,7 +17,7 @@
             <div class="form-group" ng-if="settings.extEnabled.assignments">
               <label class="col-xs-12 col-sm-3 control-label">Type:</label>
               <div class="col-xs-12 col-sm-9">
-                <ui-select prevent-animations multiple ng-model="filterParams.assignmentType">
+                <ui-select prevent-animations multiple ng-model="list.filterParams.assignmentType">
```

Comment:
That file didn't contain `List.` before PR Feedback. Did it work at all?

Reply:
No it didn't! I didnot notice the change would affect this file as well. I will fix it now.

Reply:
Could you please describe all steps/methods to ensure that all affected files were amended accordingly?

Reply:
**Steps taken:**
1. Search every instance of `List._property_` in `views/dashboard/calendar.documentList.html`, `views/dashboard/calendar.taskList.html`, `views/dashboard/documents.html`, `views/dashboard/documents.html`,  `views/contact/documents.html`,  and `views/dashboard/documents.html`.
2. Replaced with new `controllerAs` object `list._property_`
3. Verified the document created/ edited and deleted
4. Verified the task can be created/ edited and deleted

Reply:
Thank you.
1) Do these steps check if the "list" or "List" was not applied at all in templates?
2) Do these steps check **all** templates?

Reply:
1. I have used `grep -R 'List.' views/` to check if  "List." is left unnoticed and not result found!
2. I have also checked using `grep -R "controllerAs: 'List'," js/src/` if any `ControllerAs: List` left in modules and not was found.

Reply:
@bmnepali please provide all "greps" and folders these greps been run through as discussed.

Reply:
@igorpavlov above grep commands were ran from within `views/` folders that stores templates for the app.